### PR TITLE
86 - expand application of allarrays repairkit fix to all properties

### LINKED ISSUE #86: Property values returned by the API are inconsistent, as list or single value.
### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

When I did this request in production:
```
curl --get 'https://pds.nasa.gov/api/search/1/classes/collections' \
    --data-urlencode 'limit=10' \
    --data-urlencode 'q=(pds:Primary_Result_Summary.pds:processing_level eq "Raw")' \
  --data-urlencode 'fields=pds:Primary_Result_Summary.pds:processing_level' \
--header 'Accept:text/csv'

```

The response is:

```
"[Raw, Calibrated]"
"[Raw, Partially Processed, Calibrated, Derived]"
"[Raw]"
"[Raw]"
"[Raw]"
"[Raw]"
"Raw"
"Raw"
"Raw"
"Raw"
```

The brackets inside the quote should be fixed by as soon as https://github.com/NASA-PDS/registry-api/issues/381 is deployed in production.

But the fact that some values don't have brackets is unexpected since we have registry-sweeper cleaning these data.

Note that the same happens for field example "pds:Time_Coordinates/pds:start_date_time" and I guess a lot of other fields.

### 🕵️ Expected behavior

I expected [...]

### 📜 To Reproduce

1.
2.
3.
...

### 🖥 Environment Info

- Version of this software [e.g. vX.Y.Z]
- Operating System: [e.g. MacOSX with Docker Desktop vX.Y]
...

### 📚 Version of Software Used

_No response_

### 🩺 Test Data / Additional context

_No response_

### 🦄 Related requirements

🦄 #xyz

### ⚙️ Engineering Details

_No response_

### PR DESCRIPTION
**Pull request by alexdunnjpl**
## 🗒️ Summary
Per title

## ⚙️ Test Data and/or Report
Manually spot-checked produced updates 

## ♻️ Related Issues
fixes #86

### CODE REVIEW

**Comment by tloubrieu-jpl** [comment]
@alexdunnjpl @jordanpadams let's wait until breakout before we decide to merge or not.

**Review by tloubrieu-jpl — APPROVED** [review]
Validated after decision during breakout meeting today.

**Comment by alexdunnjpl** [comment]
deployed to ECR

**Pull request merged**
